openforcefield/topology/topology.py

```diff
@@ -45,9 +45,17 @@
 # GLOBAL PARAMETERS
 #=============================================================================================
 
+# TODO: Can we have the `ALLOWED_*_MODELS` list automatically appear in the docstrings below?
+# TODO: Should these be the Python equivalent of enumerated types.
+
 DEFAULT_AROMATICITY_MODEL = 'MDL' # TODO: Is there a more specific spec for this?
+ALLOWED_AROMATICITY_MODELS = ['MDL']
+
 DEFAULT_FRACTIONAL_BONDORDER_MODEL = 'Wiberg' # TODO: Is there a more specific spec for this?
-DEFAULT_CHARGE_MODEL = 'AM1-BCC' # TODO: Should this be `AM1` so that BCCs can appear in the SMIRNOFF forcefield?
+ALLOWED_FRACTIONAL_BONDORDER_MODELS = ['Wiberg']
+
+DEFAULT_CHARGE_MODEL = 'AM1-CM2' # TODO: Should this be `AM1-BCC`?
```

Comment:
I think it should be AM1-BCC by default. Then if someone wants to do BCC updates, they can go with AM1 or AM1-CM2 or... and apply their own BCCs.